domains/neodevils.json

```
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "Neodevils",
+    "email": "neo.ibrahimgunes@gmail.com"
+  },
+  "records": {
+    "CNAME": "neodevils.github.io/Neodevils/"
```

CNAME does not allow any trailing slashes (/)
```suggestion
    "CNAME": "neodevils.github.io"
```